src/components/CreateMatch/CreateMatch.js

```diff
@@ -9,20 +13,20 @@ import field from '../../images/field-option.png';
 import configure from '../../images/configure.png';
 import close from '../../images/delete-team.png';
 import shield from '../../images/shield.png';
-import Team from './Team/Team';
-import CreateTeam from './Team/CreateTeam/CreateTeam';
 import check from '../../images/check-icon.png';
 import deleteIcon from '../../images/delete-icon.png';
 import editIcon from '../../images/edit-icon.png';
 import ballPlay from '../../images/ball-play.png';
 import captain from '../../images/captain.png';
-import Users from '../../Library/Users';
+import './CreateMatch.scss';
+
 
 class CreateMatch extends Component { 
     constructor () {
```

Comment:
If you are not using props to set the state, define the state outside contructor and delete the constructor method.